pygfx/cameras/_orthographic.py

```diff
@@ -33,29 +31,35 @@ def __repr__(self) -> str:
         )
 
     def set_viewport_size(self, width, height):
-        self._view_aspect = width / height
+        self.width = float(width)
+        self.height = float(height)
+        self._view_aspect = self.width / self.height
+        self.update_bounds()
 
-    def update_projection_matrix(self):
+    def update_bounds(self):
         # Get the reference width / height
+        # TODO: when can this lead to a change? unclear to me
         width = self.width / self.zoom
         height = self.height / self.zoom
```

Comment:
if `self.zoom` is a single number, how could it ever result in a new aspect ratio?

Reply:
yes, but I'd say that we don't need to recompute the aspect ratio then, making the function a lot simpler

Reply:
I don't follow, you mean we don't need to call this method if only the zoom changes?

Reply:
we have this:

    def set_viewport_size(self, width, height):
        self.width = float(width)
        self.height = float(height)
        self._view_aspect = self.width / self.height

and this:

        width = self.width / self.zoom
        height = self.height / self.zoom
        # Increase either the width or height, depending on the view size
        aspect = width / height

and I don't see how that second `aspect` could ever be different from `self._view_aspect`